docs/src/topics/flavors/cilium-bgp-lb.md

```diff
@@ -0,0 +1,74 @@
+# Cilium BGP Load-Balancing
+
+This flavor creates special labeled worker nodes for ingress which leverage Cilium's
+[BGP Control Plane](https://docs.cilium.io/en/stable/network/bgp-control-plane/)
+and [LB IPAM](https://docs.cilium.io/en/stable/network/lb-ipam/) support.
+
+With this flavor, Services exposed via `type: LoadBalancer` automatically get
+assigned an `ExternalIP` provisioned as a shared IP through the
+[linode-CCM](https://github.com/linode/linode-cloud-controller-manager?tab=readme-ov-file#shared-ip-load-balancing),
+which is deployed with the necessary settings to perform shared IP load-balancing.
+
+```admonish warning
+There are several important caveats to load balancing support based on current
+Linode networking and API limitations:
+
+1. **Services with external IPv6 addresses are not reachable**
```

Comment:
Is this currently relevant to our implementation since we are just sharing ipv6 addresses anyways?

Reply:
I'll take this out, this was left over from an earlier implementation where the CCM didn't automatically create the shared IP. Now the user doesn't have to necessarily worry about the ipv6 gotcha for this flavor.